protobuf/coreapi/api.proto

```diff
@@ -243,10 +243,17 @@ message DeployServiceReply {
     Type type = 2;      // Type of status message.
   }
 
+  message Service {
```

Comment:
I don't like that we define the second time Service message in the same package.

In fact, this replay got quite bolted.

We should replace `validationError` with just return an error from deploy method (don't remember why we need this special kind of error).

In new #799 PR we only send 2 statuses about deploying so we can think of replacing this status api to something more general than any of the methods can use if wanted.

Applying the above issues will result in simple replay:
```
message DeployServiceReply {
  string sid = 1;   // `sid` is the service ID.
  string hash = 2;  // `hash` is the service hash.
}
```

Reply:
As agreed with Hubert, it will be improved in PR refactor